banzai_floyds/cosmics.py

```diff
@@ -0,0 +1,13 @@
+from banzai.stages import Stage
+from astroscrappy import detect_cosmics
+
+
+class CosmicRayMasker(Stage):
+    def do_stage(image):
```

Comment:
does this not need to be `(self, image)`?
it doesn't seem like this method is run anywhere or is tested yet.